pontoon/contributors/views.py

```diff
@@ -141,7 +145,10 @@ def toggle_user_profile_attribute(request, username):
         )
 
     profile = user.profile
-    setattr(profile, attribute, json.loads(value))
+    try:
+        setattr(profile, attribute, json.loads(value))
+    except json.decoder.JSONDecodeError:
+        setattr(profile, attribute, value)
```

Comment:
What's the reason for this change? It seems a bit weird to accept data as either JSON-packaged and not; this seems like a prime way of writing bad data into the db.

Reply:
`json.loads` is used to convert JS Booleans to Python Booleans. 

The newly introduced fields aren't Boolean.

I'll be more explicit about this.